nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryQuery.java

```diff
@@ -2295,6 +2293,69 @@ public void testSelectColumnsSameName() throws Exception {
         res.close();
     }
 
+    /**
+     * @since 9.3
+     */
+    @Test(expected = QueryParseException.class)
+    public void testUnsupportedColumnMappings() throws Exception {
+        //ecm:majorVersion is NOT supported
+        String query = "SELECT ecm:majorVersion FROM File";
+        IterableQueryResult res = session.queryAndFetch(query, "NXQL");
+    }
+
+    /**
+     * @since 9.3
+     */
+    @Test
+    public void testSelectColumnMappings() throws Exception {
+        String query;
+        IterableQueryResult res;
+        Map<String, Serializable> map;
+
+        DocumentModel file = new DocumentModelImpl("/", "testfileForColumns", "File");
```

Comment:
You're probably copy/pasting from other tests, but now we prefer to not rely on the implementation and do instead:

    DocumentModel file = session.createDocumentModel("/", "testfileForColumns", "File");

Reply:
Something to remember for the next time.  I will leave it how it is for this test so its consistent with the many other tests in this class.